### DISCUSSION TITLE
Homepage bug

### DISCUSSION THREAD

**Discussion by gankumangcao** [community/community#144018] [Mobile]
This bug has been there for several days，Reinstalling github still doesn't work，Request a fix。
![IMG_20241109_023013.jpg](https://github.com/user-attachments/assets/e2cc8a68-8231-4358-b84f-5c92035ea5c9)

**Comment by kangarruu** [comment]
Hi @gankumangcao thanks for writing in! Has this issue resolved for you?

**Reply by gankumangcao** [reply]
Solved

**Reply by kangarruu** [reply]
Great to hear!
